uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup/leave-request-popup-details-tab.html

```diff
@@ -84,7 +84,7 @@
 </crm-loading>
 
 <!--Expiry Date Section - Only For Toil-->
-<div class="row" ng-show="$ctrl.isLeaveType('toil')">
+<div class="row" ng-show="$ctrl.isLeaveType('toil') && $ctrl.requestCanExpire">
```

Comment:
no bug that I'm aware of. `initRequestCanExpire` has not changed (well, only the name), the one that change was the `canExpire` method, but the query is different. 

The filter is:
```
accrual_expiration_unit: { 'IS NOT NULL': 1 },
accrual_expiration_duration: { 'IS NOT NULL': 1 },
```

and the condition/result is: `return results.length > 0;` meaning: if the expiration unit and duration are not null, then it means that the record can expire.